README.md

```diff
@@ -2,7 +2,7 @@
 
 # typed-async-storage
 
-A tiny wrapper for [AsyncStorage](https://github.com/react-native-community/async-storage) that allows creating schema-based storage and validation using [PropTypes](https://www.npmjs.com/package/prop-types) 
+A tiny wrapper for [AsyncStorage](https://github.com/react-native-community/async-storage) that allows creating an unencrypted, asynchronous, persistent and schema-based storage (that is global to an app and should be used instead of LocalStorage) and validation using [PropTypes](https://www.npmjs.com/package/prop-types) 
```

Comment:
1. This change bloats the description I'm trying to keep it as short as possible
2. You can not use the package instead of LocalStorage, because it is only for react-native and you use AsyncStorage there

Reply:
Ah alright understood... it does bloat the description. True.
oh okay... I'll make the necessary changes